lib/addons/backstage/index.ts

```diff
@@ -0,0 +1,111 @@
+import { Construct } from "constructs";
+import { HelmAddOn, HelmAddOnUserProps } from "../helm-addon";
+import { dependable, setPath } from "../../utils";
+import { ClusterInfo, Values } from "../../spi";
+import { ICertificate } from "aws-cdk-lib/aws-certificatemanager";
+import * as rds from "aws-cdk-lib/aws-rds";
+import { CfnOutput } from 'aws-cdk-lib';
+
+const HTTPS = "https://";
+
+/**
+ * User provided options for the Helm Chart
+ */
+export interface BackstageAddOnProps extends HelmAddOnUserProps {
+    subdomain: string,
+    certificateResourceName: string,
+    imageRegistry: string,
+    imageRepository: string,
+    imageTag?: string,
```

Comment:
1. members of the interface must be segregated by ";" not ","
2. public members of the props that are expected to be provided by the users must have jsdoc associated with it. Example [here](https://github.com/aws-quickstart/cdk-eks-blueprints/blob/main/lib/addons/appmesh/index.ts#L14). This documentation then  becomes part of public API documentation [here](https://aws-quickstart.github.io/cdk-eks-blueprints/api/).